doc/usermanual/darkroom/panels/duplicate.xml

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE sect2 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+                <!ENTITY % darktable_dtd SYSTEM "../../dtd/darktable.dtd">
+                %darktable_dtd;
+                ]>
+<sect2 status="final" id="duplicate">
+
+  <title>Duplicate manager</title>
+
+  <indexterm>
+    <primary>darkroom panels</primary>
+    <secondary>duplicate</secondary>
+  </indexterm>
+
+  <indexterm>
+    <primary>duplicate</primary>
+  </indexterm>
+
+  <informaltable frame="none">
+    <tgroup cols="2" colsep="0" rowsep="0">
+      <colspec colwidth="6*"/>
+      <colspec colwidth="4*"/>
+      <tbody>
+        <row>
+          <entry>
+            The duplicate manager lists all versions (see duplicate <xref linkend="selected_images"/>)
+            of your image with their preview thumbnails.
+            Single click on a thumbnail to temporarily show the related version inside the center view.
```

Comment:
iirc no because it reuse the preview stored in mipmaps (the same used in lighttable for the full screen). To display zoomed preview, you need to 'compute' all the image : creating new pixelpipes etc...